api/src/services/__tests__/meetingsService.ts

```diff
@@ -110,6 +111,29 @@ describe('meetingsService', () => {
     });
   });
 
+  describe('participantExists', () => {
+    it('should query for the participant with the given id and associated meeting with given id', async () => {
+      const meetingId = 2;
+      const principalId = 3;
+      mockQuery(
+        'select `id` from `participants` where `meeting_id` = ? and `principal_id` = ?',
+        [meetingId, principalId],
+        [2]
```

Comment:
This is an invalid response for this query. This query will respond with an array of objects with `id` properties. This should be updated to respond with a value that matches with what Knex would actually return. For example:

```suggestion
        [{ id: 1 }]
```

Reply:
ok will do it now